astroquery/simbad/core.py

```diff
@@ -309,10 +309,13 @@ def list_votable_fields(self):
 
         if not hasattr(self,'_votable_fields_table'):
             # load the table
-            votable_fields_table = Table.read(
-                get_pkg_data_filename(os.path.join('data',
-                                                   'votable_fields_table.txt')),
-                format='ascii')
+            dict_file = get_pkg_data_filename(
+                os.path.join('data', 'votable_fields_dict.json'))
+            with open(dict_file, "r") as f:
+                fields_dict = json.load(f)
+            rows_to_add = list(zip(*[iter(sorted(list(fields_dict.keys())))]))
```

Comment:
Is this equivalent to `rows_to_add = [(x,) for x in fields_dict]`?  If so, I recommend using that or otherwise trying to simplify the line.  If not, please clarify what you're trying to do; I'm having a hard time parsing this line (and a little confused about what's happening with this variable in the `Table` below)